test/Oriflame.RedisMessaging.ReliableDelivery.Tests/Subscribe/MessageProcessorTest.cs

```diff
@@ -28,8 +28,10 @@ public MessageProcessorTraceable(int sleepMilliseconds, LinkedList<Message> mess
 
             public IEnumerable<Message> NewestMessages { get; set; }
 
-            protected override void HandleMessageImpl(Message message, string physicalOrLogicalChannel)
+            protected override void HandleMessageImpl(Message message, string physicalOrLogicalChannel, bool isBulkProcessing, out IMessageValidationResult validationResult)

```

Comment:
Is a parameter `isBulkProcessing` used? I am not able to see its usage in a method body.

Reply:
Yes, it is not used in test class but the declaration of method has to be kept.

Reply:
Clear. I missed that it is an override. Thank you for pointing this out!